python/ray/_private/services.py

```diff
@@ -1494,6 +1488,11 @@ def start_gcs_server(
         ]
     if redis_password:
         command += [f"--redis_password={redis_password}"]
+
+    # Logging is fully managed by C++ side spdlog, which supports rotation and file
```

Comment:
If user sets `RAY_LOG_TO_STDERR`, we don't want to set stdout to `/dev/null`

Reply:
Thanks! I didn't know that